Content.Shared/Pinpointer/MapWarpRequest.cs

```diff
@@ -0,0 +1,17 @@
+using System.Numerics;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Pinpointer;
+
+[Serializable, NetSerializable]
+public sealed class MapWarpRequest: EntityEventArgs
```

Comment:
Change event name to NavMapWarpRequest maybe? I've just got a little bit confused

Reply:
Fair point, I changed it